services/app-api/forms/mcpar.json

```diff
@@ -1271,7 +1263,7 @@
                     "validation": "radio",
                     "props": {
                       "label": "C2.V.3 Standard type",
-                      "hint": "What is the standard type? Select one or add free text if you select \"Other\".",
+                      "hint": "What is the standard type? Select one or add free text if you select “Other”.",
```

Comment:
The end of the second sentence here has a period where all the following ones with similar formats do not. Choose to have or not have a period and apply to all :)

Here through C2.V.8